web/themes/custom/hoeringsportal/templates/paragraph/paragraph--info-box.html.twig

```diff
@@ -41,7 +41,7 @@
 <div class="container">
   <div class="row">
     <div class="col-md-8">
-      <div class="info-box rounded overflow-hidden {{ elements.field_variant.0['#plain_text']|clean_class }}">
+      <div class="info-box rounded overflow-hidden {{ elements.field_variant_select.0['#markup']|clean_class }}">
```

Comment:
Can `value` be used here, i.e.

```suggestion
      <div class="info-box rounded overflow-hidden {{ elements.field_variant_select.value|clean_class }}">
```

?

Reply:
Would love to use `value` since it is cleaner. But unfortunately, it only returns `null`.